projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -135,6 +202,8 @@ export function selectAccountShippingAddress() {
     .find('.cx-summary-amount')
     .should('not.be.empty');
 
+  cy.get('cx-cost-center select').select(costCenter);
```

Comment:
force costCenter selection as we relied on default value previously. 
Default costCenter selection is different on 2211 because costCenter entities PK values are different, default value is base on PK alphabetical order.